stories/Card.stories.mdx

```diff
@@ -212,7 +248,49 @@ Neste parâmetro, enviamos o chip com as opções para o dropdown.
     }}
     decorators={[
       moduleMetadata({
-        imports: [CommonModule, IonSharedModule, IonChipModule],
+        imports: [
+          CommonModule,
+          IonSharedModule,
+          IonChipModule,
+          IonTooltipModule,
+        ],
+      }),
+    ]}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Info icon
+
+Nest paramêtro o IonCard recebe um valor booleano para renderizar o info icon.
```

Comment:
"Receive tooltip properties and show an information icon" or something more descriptive.

Reply:
corrected, thank you